db/index.js

```diff
@@ -0,0 +1,92 @@
+const db = require('mongoose');
+require('mongoose-currency').loadType(db);
+
+const { Currency } = db.Types;
+
+db.connect('mongodb://localhost:27017/ontarget', { useNewUrlParser: true });
```

Comment:
just a note - when you deploy this the uri will be different - you can set up a package like dotenv to put your keys into the env like you did for express port - https://github.com/motdotla/dotenv